src/components/card/Card.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export const PADDING = {
+  SMALL: 'small-padding',
```

Comment:
padding-small ? padding-large?

Reply:
👍

Reply:
@aju btw, we name it same way in box. should I change it there as well?